ironforgedbot/config.py

```diff
@@ -17,7 +17,7 @@ def __init__(self):
         self.BOT_TOKEN: str = os.getenv("BOT_TOKEN", "")
         self.WOM_GROUP_ID: int = int(os.getenv("WOM_GROUP_ID") or 0)
         self.WOM_API_KEY: str = os.getenv("WOM_API_KEY", "")
-        self.RANKS_UPDATE_CHANNEL: int = int(os.getenv("RANKS_UPDATE_CHANNEL") or 0)
+        self.RANKS_UPDATE_CHANNEL: str = os.getenv("RANKS_UPDATE_CHANNEL")
```

Comment:
My mistake.

Question: Would it not be better to use the channel id instead of the name? Would make it less brittle

Reply:
Pure convenience -- easier to see channel name than id